.github/workflows/checks.yml

```
@@ -224,6 +224,14 @@ jobs:
               --name chart-testing \
               "ko.local/cluster-api-runtime-extensions-nutanix:$(devbox run -- gojq -r .version dist/metadata.json)"
 
+      - if: steps.list-changed.outputs.changed == 'true'
+        name: Sideload docker image caren-helm
+        run: |
+          devbox run -- \
+            kind load docker-image \
+            --name chart-testing \
+            "ghcr.io/nutanix-cloud-native/caren-helm-reg:$(devbox run -- gojq -r .version dist/metadata.json)"
```

You can combine loading both docker images into the same step:

```suggestion
              "ko.local/cluster-api-runtime-extensions-nutanix:$(devbox run -- gojq -r .version dist/metadata.json)" \
              "ghcr.io/nutanix-cloud-native/caren-helm-reg:$(devbox run -- gojq -r .version dist/metadata.json)"
```